compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -160,6 +170,27 @@ internal class ComposeContainer(
         layers.fastForEach(DesktopComposeSceneLayer::onChangeWindowFocus)
     }
 
+    override fun windowOpened(e: WindowEvent) {
+        lifecycleRegistry.handleLifecycleEvent(Lifecycle.Event.ON_RESUME)
```

Comment:
I guess it will be missed if Compose were added after window creation (via `ComposePanel`)